docs/api.md

```diff
@@ -254,3 +254,16 @@ the `requestId` in the `monetizationstart` browser event.
 ```http
 Web-Monetization-Id: dcd479ad-7d8d-4210-956a-13c14b8c67eb
 ```
+
+## Iframes
+You can monetize an iframe by adding `monetization` to the iframe's `allow` attribute. For example:
+
+```html
+  <iframe src = "/wm/example.htm" width = "600" height = "800" allow="monetization">
+```
+
+The iframe **must** have the meta tag in its head. In the example above, `example.htm` contains the Web Monetization `meta` tag.
+
+> Monetization is only supported on iframes in the top window, not iframes within iframes.
```

Comment:
We have nested iframe support in queue:
https://github.com/coilhq/web-monetization-projects/pull/460